content/it/cloud-computing.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Cloud Computing
+status: Completed
+category: Concetto
+---
+
+## Cos'è
+Il cloud computing è un modello che offre risorse di calcolo come CPU, rete e capacità del disco su richiesta su Internet. Il cloud computing offre agli utenti la possibilità di accedere e utilizzare la potenza di calcolo in una posizione fisica remota. I provider cloud come AWS, GCP, Azure, DigitalOcean e altri offrono tutti a terze parti la possibilità di affittare l'accesso alle risorse di calcolo in più posizioni geografiche.
+
+## Quale problema affronta 
+Le organizzazioni tradizionalmente hanno dovuto affrontare due problemi principali quando tentavano di espandere l'uso della potenza di calcolo. Possono o acquistare, supportare, progettare e pagare strutture per ospitare i loro server fisici e la rete o espandere e mantenere tali strutture. Il cloud computing consente alle organizzazioni di esternalizzare parte delle proprie esigenze informatiche a un'altra organizzazione.
+
+## In che modo aiuta
+I provider cloud offrono alle organizzazioni la possibilità di noleggiare risorse di calcolo su richiesta e pagare per l'utilizzo. Ciò consente due innovazioni principali: le organizzazioni possono provare le risorse senza perdere tempo a pianificare e spendere denaro o risorse per la nuova infrastruttura fisica e possono [scalare](/scalabilità/) secondo necessità e su richiesta. Il cloud computing consente alle organizzazioni di adottare tutta l'infrastruttura di cui hanno bisogno.
```

Comment:
eh, io ci provo sempre :D. Ho tolto il link perchè non abbiamo ancora tradotto scalability.
ho aggiunto 'e sola' perchè ho visto che in inglese dice 'to adopt as much **or as little** infrastructure ...'
```suggestion
I provider cloud offrono alle organizzazioni la possibilità di noleggiare risorse di calcolo su richiesta e pagare per l'utilizzo. Ciò consente due innovazioni principali: le organizzazioni possono provare le risorse senza perdere tempo a pianificare e spendere denaro o risorse per la nuova infrastruttura fisica; inoltre possono essere scalabili secondo necessità e su richiesta. Il cloud computing consente alle organizzazioni di adottare tutta e sola l'infrastruttura di cui hanno bisogno.
```

Reply:
direi, però, "adottare l'infrastruttura necessaria, tanta o poca che sia."

Reply:
incluso nel suggerimento